src/commands/apps-create.ts

```diff
@@ -0,0 +1,210 @@
+import * as clc from "cli-color";
+import * as ora from "ora";
+
+import * as Command from "../command";
+import * as getProjectId from "../getProjectId";
+import * as FirebaseError from "../error";
+import { AppPlatform, createAndroidApp, createIosApp, createWebApp } from "../appsCreate";
+import { prompt, promptOnce, Question } from "../prompt";
+import * as requireAuth from "../requireAuth";
+import * as logger from "../logger";
+
+function promptDisplayName(): Question {
+  return {
+    type: "input",
+    name: "displayName",
+    default: "",
+    message: "What would you like to call your app? ",
+  };
+}
+
+function logPostAppCreationInformation(
+  appMetadata: {
+    appId: string;
+    displayName: string;
+    bundleId?: string;
+    packageName?: string;
+    appStoreId?: string;
+  },
+  appPlatform: AppPlatform
+): void {
+  logger.info("");
+  logger.info(`🎉🎉🎉 Your Firebase ${appPlatform} App is ready! 🎉🎉🎉`);
+  logger.info("");
+  logger.info("App information:");
+  logger.info(`  - App ID: ${appMetadata.appId}`);
+  if (appMetadata.displayName) {
+    logger.info(`  - Display name: ${appMetadata.displayName}`);
+  }
+  if (appPlatform === AppPlatform.IOS) {
+    logger.info(`  - Bundle ID: ${appMetadata.bundleId}`);
+    if (appMetadata.appStoreId) {
+      logger.info(`  - App Store ID: ${appMetadata.appStoreId}`);
+    }
+  } else if (appPlatform === AppPlatform.ANDROID) {
+    logger.info(`  - Package name: ${appMetadata.packageName}`);
+  }
+
+  // TODO(caot): Uncomment this after apps:sdkconfig command is implemented
+  // logger.info("");
+  // logger.info("Run this command to print out your config file:");
+  // logger.info(`  firebase apps:sdkconfig ${appMetadata.appId}`);
+}
+
+async function initiateIosAppCreation(options: {
+  project: string;
+  nonInteractive: boolean;
+  displayName?: string;
+  bundleId?: string;
+  appStoreId?: string;
+}): Promise<any> {
+  if (!options.nonInteractive) {
+    await prompt(options, [
+      promptDisplayName(),
+      {
+        type: "input",
+        default: "",
+        name: "bundleId",
+        message: "Please specify your iOS app bundle ID: ",
```

Comment:
Is the space on the end of this message (and the one below, and the one way above for that matter) necessary?

Reply:
Confirm that they're not needed. Removed the extra spaces